tests/unit/test.program.js

```diff
@@ -434,11 +434,15 @@ describe('program.defaultVersionGetter', () => {
       });
   });
 
-  it('returns git commit information in development', () => {
+  it('returns git commit information in development', function() {
+    if (process.env.APPVEYOR) {
+      console.log('Test skipped because of $APPVEYOR'); // eslint-disable-line no-console
```

Comment:
:+1: an inline comment which reference the related github issue will be enough ;-)